subjects/java/checkpoints/multiplication-table/README.md

```diff
@@ -0,0 +1,45 @@
+## MultiplicationTable
+
+### Instructions
+
+in a file named `MultiplicationTable.java` Write a function `generate` which takes a number as a parameter, and prints the multiplication table for this number.
```

Comment:
This sentence should start with a capital letter

Reply:
done